test/run_test.sh

```diff
@@ -54,7 +58,7 @@ mv privcount.* old/ || true
 
 # Then run the injector, ts, sk, and dc
 echo "Launching injector, tally server, share keeper, and data collector..."
-privcount inject --simulate --port 20003 --log events.txt &
+xzcat events.txt.xz | privcount inject --simulate --port 20003 --log - &
```

Comment:
I usually use `xz` by default because it is better at compression: 98k compared to 147k for gzip in this case. Good point though, I'll change to gzip since the file is small in both cases.